# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/d3d11/CompositorD3D11.h
# Commit: 91cb8ee8ddd8
# Full Hash: 91cb8ee8ddd844699c7190e3d73d65ccd92934c3
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r 1086a6f8a332 -r 91cb8ee8ddd8 gfx/layers/d3d11/CompositorD3D11.h
--- a/gfx/layers/d3d11/CompositorD3D11.h	Thu Nov 05 19:28:57 2020 +0000
+++ b/gfx/layers/d3d11/CompositorD3D11.h	Thu Nov 05 19:26:21 2020 +0000
@@ -142,6 +142,13 @@
     mAllowFrameRecording = aWillRecord;
   }
 
+  void Readback(gfx::DrawTarget* aDrawTarget) {
+    mTarget = aDrawTarget;
+    mTargetBounds = gfx::IntRect();
+    PaintToTarget();
+    mTarget = nullptr;
+  }
+
  private:
   enum Severity {
     Recoverable,